.github/workflows/basic.yml

```diff
@@ -17,14 +17,15 @@ jobs:
         pool_tracking: ['ON', 'OFF']
         shared_library: ['OFF']
         os_provider: ['ON']
+        sanitizers: [{asan: 'OFF', ubsan: 'OFF', tsan: 'OFF'}]
```

Comment:
what about leak sanitizer?

Reply:
LeakSanitizer is integrated into the AddressSanitizer and enabled by default (https://github.com/google/sanitizers/wiki/AddressSanitizerLeakSanitizer).